packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -88,7 +90,8 @@ const getDefaultComponents = (
 };
 
 const defaultTypingString = (typingUsers: CommunicationParticipant[]): string => {
-  return typingUsers.length > 0 ? (typingUsers.length > 1 ? ' are typing...' : ' is typing...') : '';
+  const typingUsersToRender = typingUsers.filter((typingUser) => typingUser.displayName !== undefined);
```

Comment:
Similar to above, should this continue to use `typingUsers` because participants without a `displayName` are still typing ;)

Reply:
It was filtered outside of this component previously, I just moved the code into the function to make code more clear, it is still sharing the same logic
In previous design, only people with displayname will be shown in the typing indicator like
A, B are typing, this count is used for determine whether it is 'are typing' or 'is typing'